src/bundle/EventListener/RequestAttributesListener.php

```diff
@@ -73,6 +92,26 @@ function (Repository $repository) use ($locationId) {
         return $location;
     }
 
+    /**
+     * @param mixed $contentId
+     * @param string $language
+     *
+     * @return Content
+     *
+     * @throws UnauthorizedException
+     * @throws NotFoundException
+     */
+    private function loadContent($contentId, string $language): Content
+    {
+        $content = $this->repository->sudo(
```

Comment:
Are you aware that using `sudo` here will load the content ignoring user permissions?